src/components/footer.jsx

```
@@ -5,30 +5,30 @@
-    <footer className='footer'>
-      <div className='footer--container'>
-        <div className='footer--right'>
-        <img src={BannerPhoto} className='footer--image' />
-          <div>
-              <p>     Developed using the <a href='https://thecatapi.com/' target="_blank">CAT API</a>, this website celebrates our feline companions in every whisker. Harnessing the power of the CAT API, we bring you a purr-fectly curated experience, from delightful images to invaluable information, all dedicated to the world of cats. Meow-vel at the wonders of technology coupled with our undying love for our whiskered friends. Tailored with care and devotion, this website is a tribute to the CAT API and the enchanting universe of cats.</p>
+    <footer className='footer bg-gray-100 shadow-lg rounded-2xl p-5'>
+      <div className='footer--container grid grid-cols-1 md:grid-cols-2 gap-4'>
+        <div className='footer--right flex flex-col items-center'>
+          <img src={BannerPhoto} className='footer--image w-full max-w-md' />
+          <div className='text-center mt-4'>
+            <p>Developed using the <a href='https://thecatapi.com/' target="_blank" className='text-blue-500 hover:underline'>CAT API</a>, this website celebrates our feline companions in every whisker. Harnessing the power of the CAT API, we bring you a purr-fectly curated experience, from delightful images to invaluable information, all dedicated to the world of cats. Meow-vel at the wonders of technology coupled with our undying love for our whiskered friends. Tailored with care and devotion, this website is a tribute to the CAT API and the enchanting universe of cats.</p>
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)